src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -107,20 +107,13 @@
 namespace U2 {
 
 /** Converts list of label args into QStringList. */
-static QStringList labels(const QString &label1, const QString &label2 = "", const QString &label3 = "", const QString &label4 = "", const QString &label5 = "") {
-    QStringList rawLabelList = QStringList() << label1 << label2 << label3 << label4 << label5;
-    QStringList resultLabelList;
-    for (const QString &rawLabel : rawLabelList) {
-        if (!rawLabel.isEmpty()) {
-            resultLabelList << rawLabel;
-        }
-    }
-    return resultLabelList;
+static QStringList labels(const QStringList &labelList) {
```

Comment:
Do you really need this function now?

Reply:
When I pass {label1, label2, label3} to macro I see a compiler error about the wrong number of params.

Reply:
Ok.